src/core/text/TextStyle.js

```diff
@@ -284,6 +284,7 @@ export default class TextStyle
         if (this._fill !== outputColor)
         {
             this._fill = outputColor;
+            this._gradient = null;
```

Comment:
First, there's the issue that the gradient could also be effected by the fillGradientStops and also fillGradientType changing

Second, it doesn't feel right that the text style class, which describes how a style should look, is suddenly holding information that the Text class has generated (the gradient). I feel that the Text class itself should store any previously generated fillGradient. 

Could it work where we have a this.fillStyleID that sits along a styleID? The text class could check against that fillStyleID having changed, and know if to re-generate a gradient that way?

Reply:
use fillStyleID/styleID will make the code become complex.
Now juse 3 places would call `this._gradient = null`.

I think in the future , if there are more places will call it , 
We could try to refactor and use   `fillStyleID/styleID`.